process/smartContract/builtInFunctions/esdtTransfer.go

```diff
@@ -120,6 +120,7 @@ func (e *esdtTransfer) ProcessBuiltinFunction(
 		}
 
 		if isSCCallAfter {
+			vmOutput.GasRemaining = vmInput.GasProvided - e.funcGasCost
```

Comment:
Could be negative in some edge cases? safeSub ?

Reply:
I do not think so. but added safeSub